<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chinese Text to Video (CnT2V) for Intel Core Ultra7</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Video:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_outputvideo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>个人赛道-王祥福</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pause">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>English Text:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_en">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Chinese Input:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_cn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tslonly">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate Only (C-&gt;E)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bycn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>By Chinese</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_byen">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>By English</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_resolution">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Resolution Set( W x H )</string>
    </property>
    <widget class="QSpinBox" name="spinBox_res_w">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>240</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
     <property name="singleStep">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>35</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_res_h">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>240</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
     <property name="singleStep">
      <number>8</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Video Duration(S)</string>
    </property>
    <widget class="QSpinBox" name="spinBox_vd">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Inference Steps</string>
    </property>
    <widget class="QSpinBox" name="spinBox_step">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>25</number>
     </property>
     <property name="value">
      <number>40</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Accelerator Selection</string>
    </property>
    <widget class="QRadioButton" name="radioButton_gpunup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Intel iGPU&amp;&amp;NPU</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_noacc">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>No Accelerator</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_export_compile">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Export IR &amp;&amp; Compile</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
